aws_adfs/login.py

```diff
@@ -151,7 +155,7 @@ def _authenticate(config, username=None, password=None):
     del password
 
     # Decode the response
-    html = ET.fromstring(response.text.decode('utf8'), ET.HTMLParser())
+    html = ET.fromstring(response.text, ET.HTMLParser())
```

Comment:
Leaving the decode('utf8') in breaks Python 3 support.
```
  File "/home/ec2-user/python34-env/lib/python3.4/site-packages/aws_adfs/login.py", line 72, in login
    principal_roles, assertion = _authenticate(config)
  File "/home/ec2-user/python34-env/lib/python3.4/site-packages/aws_adfs/login.py", line 158, in _authenticate
    html = ET.fromstring(response.text.decode('utf8'), ET.HTMLParser())
AttributeError: 'str' object has no attribute 'decode'
```

Removing it works fine for me with both Python 2.7 and 3.4; are you seeing something else?